src/pygeoprocessing/routing/routing.pyx

```diff
@@ -611,7 +607,9 @@ def _generate_read_bounds(offset_dict, raster_x_size, raster_y_size):
 
 def fill_pits(
         dem_raster_path_band, target_filled_dem_raster_path,
-        working_dir=None, long max_pixel_fill_count=_MAX_PIXEL_FILL_COUNT,
+        working_dir=None,
+        long long max_pixel_fill_count=-1,

```

Comment:
This is probably going to have an impact on InVEST models if the default was `500`. Is there an issue in the InVEST repo mentioning this change and the need to update the models?

Reply:
Sure, I made an issue here: https://github.com/natcap/invest/issues/560 won't affect any test data and it was a stopgap to only affect runs that were run on degenerate DEMs.